public/usage-examples/graphics/fill_rectangle_on_window/fill_rectangle_on_window-1-simple-oop.cs

```diff
@@ -0,0 +1,19 @@
+using SplashKitSDK;
+
+namespace Program
```

Comment:
Update the namespace to be more descriptive of the usage example. e.g FillRectangleOnWindowExample